main.py

```diff
@@ -153,7 +153,9 @@ def funHideout(inputNum):
     
     inputSecondNum = getInputNumber()
 
-    CraftingTime = temp[makingKeys[inputNum-1]][makingValues[inputSecondNum-1]]
+    craftingTime = temp[makingKeys[inputNum-1]][makingValues[inputSecondNum-1]]
```

Comment:
```suggestion
    craftingTime = temp[makingKeys[inputNum - 1]][makingValues[inputSecondNum - 1]]
```

보통은 이렇게 연산자는 양쪽 사이 띄워줍니다!